.circleci/config.yml

```diff
@@ -484,7 +484,7 @@ workflows:
             - gcp context
           filters:
             branches:
-              ignore: /^(pull\/).*$/
+              ignore: /^(pull|dependabot)\/.*$/
```

Comment:
this prevents dependabot complaining on outdated dependencies for dependabot-created branches?